servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSHandler.java

```diff
@@ -96,4 +96,6 @@
     DomainRoleMembership getPendingDomainRoleMembersList(ResourceContext context, String principal);
     Schema getRdlSchema(ResourceContext context);
     ResourceContext newResourceContext(HttpServletRequest request, HttpServletResponse response);
+    void recordMetrics(String httpMethod, int httpStatus, String apiName, ResourceContext ctx);
+    Object startMetricTiming();
```

Comment:
not sure why we need this? we should be able to hide the constructions of the metric within the newResourceContext api call so we don't need to expose this